src/components/Input/Input.js

```diff
@@ -400,7 +448,7 @@ StyledInput.propTypes = Input.propTypes;
 
 Input.defaultProps = {
   children: null,
-  element: 'input',
+  as: 'input',
```

Comment:
Are you sure we should REPLACE the default props? This could be a breaking change, no? If someone was relying on this?

Reply:
We still use `HtmlElement` and its `element` prop under the hood, but map `as` to it: https://github.com/sumup/circuit-ui/pull/445/files#diff-8030fbcffcceb6c2680e6536fb739561R310

So it's not a breaking change :)

Reply:
Is this also handled by one of the previous tests?

Just want to make sure there are no surprises on a component as important as `Input`!

Reply:
It is verified by an existing test: https://github.com/sumup/circuit-ui/pull/445/files#diff-57684e79688c4508829b0d077ff049acR28

Reply:
Hm, I get what you're doing, but shouldn't we still have the `element` prop default value specified and ignore it as long as the `as` has the same or a different value? Hm you probably cannot provide a default value when at the same time you want to provide a deprecation warning, right? 😆

Reply:
> You probably cannot provide a default value when at the same time you want to provide a deprecation warning, right?

Exactly.